Tools/MGCB/ConsoleLogger.cs

```diff
@@ -11,13 +11,13 @@ public class ConsoleLogger : ContentBuildLogger
     {
         public override void LogMessage(string message, params object[] messageArgs)
         {
-			Console.WriteLine(message, messageArgs);
+			Console.WriteLine(IndentString + message, messageArgs);
```

Comment:
We write some exceptions directly to Console.
https://github.com/mono/MonoGame/blob/develop/Tools/MGCB/BuildContent.cs#L315-L342
We have to change this to Logger.LogImportantMessage()